Maestro/subscriptions.json

```diff
@@ -689,6 +689,23 @@
         }
       }
     },
+    // Automate merging aspnetcore release/6.0-rc branches back to release/6.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/aspnetcore/blob/release/6.0-rc/**/*"
```

Comment:
Wondering, doesn't that need a star after -rc as well? `/6.0-rc*/`. Asking to make sure https://github.com/dotnet/versions/pull/775 is right.

Reply:
Nope, the syntax is confusing and isn't a glob pattern.